db/schema.rb

```diff
@@ -291,6 +303,26 @@
 
   add_index "searches", ["user_id"], name: "index_searches_on_user_id", using: :btree
 
+  create_table "topic_translations", force: :cascade do |t|
```

Comment:
@lbiedinger The schema committed here includes work from another branch for the topics and categorisations but which has not yet been merged. Could you remove these?